src/mlpack/CMakeLists.txt

```diff
@@ -158,3 +159,62 @@ endif ()
 # If we are building Markdown documentation, we have to run some setup after we
 # recurse into methods/.  If not, this function is empty.
 post_markdown_setup()
+
+# Installing CMake Helper Files
```

Comment:
Maybe it's worth changing this comment to point out that this block of code is focused on generating `mlpack-config.cmake` and related files?

Reply:
```suggestion
# Installing CMake Helper Files for mlpack-config.cmake
```

Reply:
These are both minor changes, I'll just add them then approve/merge.